src/EditorFeatures/TestUtilities/Workspaces/TestForegroundNotificationService.cs

```diff
@@ -27,7 +27,7 @@ public void RegisterNotification(Func<bool> action, int delayInMS, IAsyncToken a
             Task task;
             lock (_gate)
             {
-                task = _queue.ScheduleTask(() => Execute_NoLock(action, asyncToken, cancellationToken), cancellationToken);
+                task = _queue.ScheduleTaskInProgress(() => Execute_NoLock(action, asyncToken, cancellationToken), cancellationToken);
```

Comment:
curious how i should read `ScheduleTaskInProgress`.  it doesn't have obvious semantics to me as to what that means.